addon/components/power-select.ts

```diff
@@ -413,7 +421,7 @@ export default class PowerSelect extends Component<PowerSelectArgs> {
     if (this.args.scrollTo) {
       return this.args.scrollTo(option, select);
     }
-    let optionsList = document.querySelector(`[aria-controls="ember-power-select-trigger-${select.uniqueId}"]`) as HTMLElement;
+    let optionsList = document.querySelector(`#ember-power-select-options-${select.uniqueId}`) as HTMLElement;
```

Comment:
If querying for an ID, maybe use `document.getElementById`

Reply:
Updated 👍